opencensus/stats/exporters/prometheus_exporter.py

```diff
@@ -356,12 +343,3 @@ def view_name(namespace, view):
     if namespace != "":
         name = namespace + "_"
     return name + view.name
-
-
-def view_signature(namespace, view):
```

Comment:
IMO this is not needed. `View` and `ViewData` can be directly indexed by its name since view names are unique.